src/OrleansAzureUtils/Providers/Storage/AzureTableStorage.cs

```diff
@@ -107,9 +109,10 @@ public Task Init(string name, IProviderRuntime providerRuntime, IProviderConfigu
 
             isDeleteStateOnClear = config.Properties.ContainsKey(DeleteOnClearPropertyName) &&
                 "true".Equals(config.Properties[DeleteOnClearPropertyName], StringComparison.OrdinalIgnoreCase);
-
-            Log = providerRuntime.GetLogger("Storage.AzureTableStorage." + id);
-
+            var loggerFactory = providerRuntime.ServiceProvider.GetRequiredService<ILoggerFactory>();
+            var loggerName = this.GetType().FullName + id;
```

Comment:
Maybe just suffix it with `-{Name}`. This `id` thing doesn't make much sense. Also, I would do a similar thing in AzureBlobStorage. Regardless of the approach, they should probably do the same thing

Reply:
BTW, you partially addressed this. We would still need to do the same thing for AzureBlobStorage

Reply:
I did it for AzureBlobStorage

Reply:
You are right, a full refresh in CodeFlow revealed it, sorry 'bout that :P